src/tabbar.cpp

```diff
@@ -28,7 +28,6 @@ TabBar::TabBar(QWidget *parent) :
                 auto current = this->currentWidget();
                 auto widget = this->createNewTab(true);
                 QUITIFNULL(current);
```

Comment:
If we do nothing with `widget` and `current` we should not get them and store them in a variable.

The whole function can be reduced to `this->createNewTab(true);`